### PR TITLE
Mobile : Fixes #8517: Fixed code block not default line wrap in pdf view

### LINKED ISSUE #8517: Mobile: pdf view of code block can't line wrap
## Environment

Joplin version:  latest code of dev branch (ffc40b27974d540662212eb6a49a15e0a8e6a301)
Platform: Android,IOS
OS specifics:  

## Steps to reproduce

1. mobile version can't make code block line wrap
2.  I have attempted to modify packages/app mobile/components/NoteEditor/CodeMirror/theme.ts :
```
'& .cm-codeBlock': {
...
			whiteSpace: 'pre-wrap',		
},
```
but this don't work, debug on Android and IOS , find out this only can affect edit view, not pdf view.

## Describe what you expected to happen

code block default line wrap

### PR DESCRIPTION
**Pull request by wljince007**
Find a way to fix #8517  

[PR #8391](https://github.com/laurent22/joplin/pull/8391) can't reopen, so I create a new pr.

### CODE REVIEW

**Comment by wljince007** [comment]
Iphone and Android test ok.

image

image

**Comment by laurent22** [comment]
Would you mind asking on the forum whether people would be fine with this change or not? I don't mind too much either way, but I wouldn't want to add this and have to revert it next month if too many users are complaining about it

**Comment by wljince007** [comment]
Ok, I well trying to open a forum to asking  whether people would be fine with this change or not, thanks a lot.

**Comment by wljince007** [comment]
add in  https://discourse.joplinapp.org/t/do-you-support-code-block-default-line-wrap-in-pdf-view-in-mobile-app/32119

File: packages/app-mobile/components/NoteBodyViewer/hooks/useSource.ts [hunk 1]
```diff
@@ -193,6 +193,15 @@ export default function useSource(noteBody: string, noteMarkupLanguage: number,
 				:root > body {
 					padding: 0;
 				}
+
+				code {
+					white-space: pre-wrap;
+				}
```
Comment (laurent22) [inline packages/app-mobile/components/NoteBodyViewer/hooks/useSource.ts]: Remove this please

File: packages/app-mobile/components/NoteBodyViewer/hooks/useSource.ts [hunk 2]
```diff
@@ -203,7 +212,7 @@ export default function useSource(noteBody: string, noteMarkupLanguage: number,
 						<meta charset="UTF-8">
 						<meta name="viewport" content="width=device-width, initial-scale=1">
 						<style>
-							${shim.mobilePlatform() === 'ios' ? iOSSpecificCss : ''}
+							${shim.mobilePlatform() === 'ios' ? iOSSpecificCss : defaultCss}
```
Comment (laurent22) [inline packages/app-mobile/components/NoteBodyViewer/hooks/useSource.ts]: Here make it `${shim.mobilePlatform() === 'ios' ? iOSSpecificCss + '\n' + defaultCss: defaultCss}`

**Comment by laurent22** [comment]
Thanks for asking on the forum. There was only one answer but a positive one and it feels like this is a good change overall, so let's merge it. Could you please just make the suggested change? This is just to reduce duplicate code

**Comment by wljince007** [comment]
Sorry for the delay.
Thank you for your patient guidance. I have made the necessary changes as you suggested. 
I sincerely hope that Joplin will become better and better and  more widely used.

**Comment by wljince007** [comment]
@laurent22  I have  modify the code as your guidance, please merge this PR.

**Comment by laurent22** [comment]
Thanks for implementing this!

**Pull request merged**